transmogrifier/sources/json/aardvark.py

```diff
@@ -56,25 +56,27 @@ def get_optional_fields(self, source_record: dict) -> dict | None:
         Overrides metaclass get_optional_fields() method.
 
         Args:
-            xml: A BeautifulSoup Tag representing a single Datacite record in
-                oai_datacite XML.
+            source_record: A JSON object representing a source record.
         """
         fields: dict = {}
 
         # alternate_titles
+        fields["alternate_titles"] = self.get_alternate_titles(source_record) or None
```

Comment:
I'm liking this pattern.  Though I think there could be value in the per-field method as we've discussed, this is explicit and easy to scan here.